src/image.c

```diff
@@ -179,12 +179,20 @@ static void wolfBoot_verify_signature(uint8_t key_slot,
 }
 #endif /* WOLFBOOT_SIGN_ECC256 */
 
+
 #if defined(WOLFBOOT_SIGN_RSA2048) || \
         defined (WOLFBOOT_SIGN_RSA3072) || \
         defined (WOLFBOOT_SIGN_RSA4096)
 #include <wolfssl/wolfcrypt/asn.h>
 #include <wolfssl/wolfcrypt/rsa.h>
 
+#if defined(WOLFBOOT_SIGN_RSA4096) && \
+    (defined(USE_FAST_MATH) && \
+     !defined(WOLFSSL_SMALL_STACK) && !defined(WOLFBOOT_HUGE_STACK))
```

Comment:
Recommend to remove `!defined(WOLFSSL_SMALL_STACK) &&` .  Seems like if small_stack IS defined OR if huge_stack is NOT defined then you should error out with the note.   I see above where both small and huge cannot be both set, so this logic will work provided the above guarantees only one of them is set.  Perhaps we should consider stack_size=1,2,3, for small, normal, huge?

BTW.  This is exactly what was causing the STM32C0 to fail at runtime due to only having 12kB of RAM.  :)